docs/bouwblokken/publieke-inkoop/index.md

```diff
@@ -1,77 +1,141 @@
 ---
 title: Publieke inkoop
```

Comment:
Algemene feedback:
In de Preview zie ik dat de site er een stuk 'cleaner' uit ziet. Dat is mooi! Toch ook wel een kritische kanttekening erbij. 

De informatieve stukken waarvan ik weet/heb getoetst bij de doelgroep dat het relevante passages zijn, zijn nagenoeg allemaal verloren gegaan. Het is nu geen verbindend schrijven meer van de relevante elementen. Ik zet dus mij vraagtekens bij of het nu nog informatief is en meer inzichten aanreikt dan lezers al hadden.

Daarbij natuurlijk ook het inzicht/erkenning dat hetgeen er stond bondiger kan worden beschreven. Ik heb alleen het gevoel er niet is herschreven maar enkel gestreept. De tekst die er stond is ook getoetst bij de Werkgroep waar zij inhoudelijk achter staan. 

Volgens mij verschillen de belevingen hiervoor nog wat teveel om verwerkt te krijgen in deze release en moeten we koersen op de volgende release.